### PR TITLE
Add ability to run tests in a Docker container

### PR DESCRIPTION
**Pull request by madelinecameron**
## Proposed Changes
  - Adds necessary config for running tests in a Docker container

### CODE REVIEW

**Comment by madelinecameron** [comment]
Also going to write a guide to explain this. I think this could be a really, really interesting technical blog post so going to write one of those to help others figure out how to do this easily. :)

File: scripts/setup [hunk 1]
```diff
@@ -0,0 +1,40 @@
...
+  cd ${repo}
+  npm install
+  npm link
+  cd ..
+done
+
+mkdir -p /root/.ara/keyrings
+
+# Create identity
+echo "test" | aid create
```
Comment (vipyne) [inline scripts/setup]: what does the `echo "test"` do here?

File: .travis.yml [hunk 2]
```diff
@@ -1,11 +1,18 @@
+sudo: required
 language: node_js
 node_js:
-  - "8"
+- 8.9.4
```
Comment (vipyne) [inline .travis.yml]: this is bc docker?

Comment (madelinecameron) [reply .travis.yml]: Unfortunately not. 

It's travis being overzealous and me giving up on fighting. 

I typically try to make my changes as directed as possible but travis changes .travis.yml back every time you run the encrypt-key command so I finally gave up. :upside_down_face:

File: scripts/setup (hunk 1 above)
Comment (madelinecameron) [reply scripts/setup]: It sets the password of the identity to "test". :)

Comment (vipyne) [reply scripts/setup]: ahhhh, TIL. 🙌

File: .travis.yml (hunk 2 above)
Comment (vipyne) [reply .travis.yml]: roger that.  we should put something in the README about that (in the docker branch 👍 )

**Pull request closed without merging**